### PR TITLE
diversify camel, lower and upper case conversions

### PR DESCRIPTION
**Pull request by pfernique**

### CODE REVIEW

File: core/src/core/util.py [hunk 1]
```diff
@@ -25,21 +27,100 @@ def camel_case_to_lower(name):
...
+def camel_case_to_upper(name):
+    """
+
+    :Examples:
+
+    >>> camel_case_to_upper('squareRoot')
+    'SQUARE_ROOT'
+
+    >>> camel_case_to_upper('SquareRoot')
+    ''SQUARE_ROOT
```
Comment (gbaty) [inline core/src/core/util.py]: I think it is 'SQUARE_ROOT'

File: core/src/core/util.py [hunk 2]
```diff
@@ -25,21 +27,100 @@ def camel_case_to_lower(name):
...
+
+    >>> camel_case_to_upper('ComputeSQRT')
+    ''COMPUTE_SQRT
```
Comment (gbaty) [inline core/src/core/util.py]: Same here ''COMPUTE_SQRT -> 'COMPUTE_SQRT'

File: core/src/core/util.py (hunk 1 above)
Comment (pfernique) [inline core/src/core/util.py]: Done

File: core/src/core/util.py (hunk 2 above)
Comment (pfernique) [inline core/src/core/util.py]: Done

**Pull request merged**